src/pages/hero/Hero.jsx

```diff
@@ -0,0 +1,204 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { Link as LinkToID } from "react-scroll";
+import StarRatings from "react-star-ratings";
+import { Footer, Navbar } from "../../components/components-index.js";
+import "./Hero.css";
+
+export const Hero = (props) => {
+  const { productData, offerData } = props;
+  const [slider, setSlider] = useState("slide1");
+
+  useEffect(() => {
+    const imgArr = ["slide1", "slide2", "slide3"];
+    const getRandomImg = setInterval(() => {
+      const random = Math.floor(Math.random() * imgArr.length);
+      setSlider(imgArr[random]);
+    }, [5000]);
+    return () => {
+      clearInterval(getRandomImg);
+    };
+  }, []);
+
+  return (
+    <>
+      <Navbar />
+      <main>
+        <div className="wrapper_main">
+          <div className="banner flex">
+            <img src={`/assets/${slider}.webp`} alt="slider1" />
+            <div className="text_cl overlay_text kodchasan">
+              <div>360°</div>
+              <div>re-imaging the world</div>
+              <div>
+                3d virtual and augmented reality panoramas
+                <br /> to demonstrate
+              </div>
+              <LinkToID to="head" offset={-50}>
+                <button className="btn btn-cta">shop now</button>
+              </LinkToID>
+            </div>
+          </div>
+          <div className="banner img_curve">
+            <img className="" src="/assets/curve1.webp" alt="curve" />
+          </div>
+          <div className="text-center kodchasan hero_title text_cl">
+            <div className="h4">High-Quality VR </div>
+            <h2 className="h2">
+              <b>The Real World Immersion</b>
+            </h2>
+          </div>
+          {offerData.map((offer) => (
```

Comment:
Suggestion: these `divs` can be extract as components

Reply:
Done!